pyvista/examples/downloads.py

```diff
@@ -2899,8 +3129,45 @@ def download_mars_jpg():  # pragma: no cover
 
     Examples
     --------
+    Download the Mars JPEG and map it to spherical coordinates on a sphere.
+
+    >>> import math
+    >>> import numpy
+    >>> import numpy as np
     >>> from pyvista import examples
-    >>> dataset = examples.download_mars_jpg()  # doctest:+SKIP
+    >>> import pyvista
+
+    Download the JPEGs and convert the Mars JPEG to a texture.
+
+    >>> mars_jpg = examples.download_mars_jpg()
+    >>> mars_tex = pyvista.read_texture(mars_jpg)
+    >>> stars_jpg = examples.download_stars_jpg()
+
+    Create a sphere mesh and compute the texture coordinates.
+
+    >>> sphere = pyvista.Sphere(radius=1, theta_resolution=120, phi_resolution=120,
+    ...                         start_theta=270.001, end_theta=270)
+    >>> sphere.active_t_coords = numpy.zeros((sphere.points.shape[0], 2))
+    >>> sphere.active_t_coords[:, 0] = 0.5 + np.arctan2(-sphere.points[:, 0],
+    ...                                                 sphere.points[:, 1])/(2 * math.pi)
+    >>> sphere.active_t_coords[:, 1] = 0.5 + np.arcsin(sphere.points[:, 2]) / math.pi
```

Comment:
I can see a few issues in these last few unreviewed commits, I'll fix them in a follow-up. @akaszynski quick question here: why didn't you use our `texture_map_to_sphere` helper? https://docs.pyvista.org/examples/02-plot/texture.html#spherical-texture-coordinates